### ISSUE TITLE
AttributeError: 'MaxPool2DBackward' object has no attribute 'kernel_size'

### ISSUE THREAD

**Issue by lzhaozi** [xuyan1115/pytorch2caffe#2]
My python env is Python 2.7.14 and PyTorch 0.3.1.
I try to convert my model, but face this problem:
AttributeError: 'MaxPool2DBackward' object has no attribute 'kernel_size'

Is it because this repo does not support PyTorch 0.3.1?

**Comment by xuyan1115** [comment]
yep, the interfaces in PyTorch 0.3.0 are different from other versions.
